configs/components/marionette-collective.rb

```diff
@@ -66,7 +66,9 @@
   when "aix"
     pkg.install_service "resources/aix/mcollective.service", nil, "mcollective"
   when "windows"
-    puts "Service files not enabled on windows"
+    # Note - this definition indicates that the file should be filtered out from the Wix
+    # harvest. A corresponding service definition file is also required in resources/windows/wix
+    pkg.install_service "SourceDir\\ProgramFiles64Folder\\PuppetLabs\\Puppet\\puppet\\bin\\rubyw.exe"
```

Comment:
It's not always going to be `ProgramFiles64Folder`. https://github.com/puppetlabs/puppet-agent/blob/master/configs%2Fprojects%2Fpuppet-agent.rb#L19